test/coding_test.exs

```diff
@@ -0,0 +1,32 @@
+defmodule MultipassEx.CodingTest do
+  @moduledoc false
+  use ExUnit.Case
+  use ExUnitProperties
+
+  import MultipassEx.TestData
+
+  property "encode/1 returns mutated string with values replaced" do
+    check all string <- pseudo_encrypted_string() do
+      url_safe = MultipassEx.Coding.encode(string)
+
+      refute String.contains?(url_safe, "+")
+      refute String.contains?(url_safe, "\n")
+      refute String.contains?(url_safe, "/")
+    end
+  end
+
+  property "decode/1 returns encoded string with mutations removed" do
+    check all string <- pseudo_encrypted_string(),
+              url_safe = MultipassEx.Coding.encode(string) do
```

Comment:
aaron pointed out to me on one of my PRs that this line should probably be moved into the body, since it's not strictly required for generation